src/QirRuntime/test/Standalone/InputTypes/qir-driver.cpp

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include <iostream>
+#include "../Shared/CLI11.hpp"
```

Comment:
where is this file coming from? do we have to take this dependency?

Reply:
Is this what provides `CLI::App`?

Reply:
This file is coming from [CLI11](https://github.com/CLIUtils/CLI11).

It is not required in the sense that we could implement our own parser to not take this dependency but I don't think that would be a good decision since we can use something that already exists, it is in actively maintained, and has all the features that we would need.

The [install](https://github.com/CLIUtils/CLI11#install) section of the _readme_ describes that one of the ways to "install" the library is by copying the header in your include folder, which I found the most convenient for experimentation. We can add it as a git submodule as an alternative.

Reply:
Moved the _CLI11.hpp_ file to the _externals_ folder and added the component to _cgmanifest.json_. We don't have to add it to the Third-Party notice since we are not distributing it through any of our packages yet. Regarding the copy of the file, I still think the header-only distribution of this library is the most convenient one so I kept a copy of the most recent release of the header.